lib/block_keys/mnemonic.ex

```diff
@@ -30,6 +30,26 @@ defmodule BlockKeys.Mnemonic do
     |> mnemonic()
   end
 
+  @doc """
+  Generates the 24 manmonic words from a given entropy
+
+  ## Examples
+
+      iex> BlockKeys.Mnemonic.generate_phrase_from_enthropy()
+      BlockKeys.Mnemonic.generate_phrase_from_enthropy("1112")
+      "couple maze erosion"
+  """
+  def generate_phrase_from_enthropy(enthropy) do
```

Comment:
I would suggest we accept a default parameter in `generate_phrase/0` in order to avoid repeating the same steps in both functions. What do you think ?

```
 def generate_phrase(entropy \\ nil) do
    case entropy do
      nil -> :crypto.strong_rand_bytes(32)
      _ -> entropy
    end
    |> entropy_hash()
    |> extract_checksum()
    |> append_checksum()
    |> :binary.bin_to_list()
    |> Enum.map(fn byte -> to_bitstring(byte, @pad_length_mnemonic) end)
    |> Enum.join()
    |> mnemonic()
  end
```

Reply:
Ok done. Could you please have a glance if the new implementation makes sense @tzumby ?

Reply:
Thanks, that's awesome!